app.py

```diff
@@ -1,11 +1,20 @@
-from flask import Flask
+from flask import Flask, render_template
 
 app = Flask(__name__)
 
 
 @app.route("/")
-def hello_world():
-    return "<p>Hello, World!</p>"
+def index():
+    sensor_data = {
+        'temperature': 22.5,
+        'humidity': 60
+    }
+
+    temperature = sensor_data.get('temperature')
+    humidity = sensor_data.get('humidity')
+    return render_template(
+    'index.html', temperature=temperature, humidity=humidity
```

Comment:
To make the linter check pass you need to do the following:
Fix indentation for the render_template call here (lines 16–17) so the arguments align properly